app/assets/stylesheets/shared/_boxed_group.scss

```diff
@@ -0,0 +1,34 @@
+/* Blatently taken from Github's settings page */
+.boxed-group {
+  position: relative;
+  border-radius: 3px;
+  margin-bottom: 30px;
+
+  h3 {
+    background-color: #f5f5f5;
+    margin: 0;
+    border-radius: 3px 3px 0 0;
+    border: 1px solid #d8d8d8;
+    border-bottom: 0;
+    padding: 9px 10px 10px;
+    font-size: 1.5em;
+    display: block;
+  }
+
+  &.dangerzone > h3 {
+    background-color: #df3e3e;
```

Comment:
Color literals like `#df3e3e` should only be used in variable declarations; they should be referred to via variable everywhere else.